### PR TITLE
Setup initial project skeleton

### PR DESCRIPTION
**Pull request by brettlangdon**
This first PR is just to setup the basic skeleton of the gem. I wanted to keep this PR separate from the actual code of the gem so we can focus on the gem structure here. I mostly used https://github.com/dburkes/webmock-disabler and https://github.com/bblimke/webmockas references here.

Contained here:
- Add `gemspec` file
- Add skeleton `lib` and `spec` directories
- Setup `RuboCop` and `RSpec` with `rake`
- Setup `travis-ci`

/cc @twolfson

### CODE REVIEW

**Comment by brettlangdon** [comment]
Build failed due to dependency requirements of `MRI >= v1.9.3`

`rainbow` requires `>=1.9.2` and `rubocop` needs `>=1.9.3`. Going to update the `.travis.yml` to reflect this.

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+language: ruby
+# Enable new container infrastructure
```
Comment (twolfson) [inline .travis.yml]: It's not really "new" any more =P

File: lib/webmock/fixtures/version.rb [hunk 2]
```diff
@@ -0,0 +1,6 @@
+module WebMock
+  # WebMock fixture helper
+  module Fixtures
+    VERSION = "0.1.0"
```
Comment (twolfson) [inline lib/webmock/fixtures/version.rb:4]: Interesting way to set up a version

File: webmock-fixtures.gemspec [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  s.homepage = "http://github.com/underdogio/webmock-fixtures"
+  s.summary = "Manage WebMock fixtures"
+  s.description = "Library to help manage WebMock fixtures"
+  s.license = "MIT"
+
+  # Dependencies
+  s.add_runtime_dependency("webmock", [">= 1.0.0", "< 2.0.0"])
+  s.add_development_dependency("rake", [">= 10.0.0", "< 11.0.0"])
+  s.add_development_dependency("rspec", ["=3.3.0"])
+  s.add_development_dependency("rubocop", ["=0.34.2"])
```
Comment (twolfson) [inline webmock-fixtures.gemspec]: We are specifying dependencies here and not in the `Gemfile`?

File: webmock-fixtures.gemspec [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+  # Files
+  s.files = `git ls-files`.split("\n")
+  s.test_files = `git ls-files -- {test,spec,features}/*`.split("\n")
```
Comment (twolfson) [inline webmock-fixtures.gemspec:24]: I'm pretty sure we don't upload a `git` repo to RubyGems. Are these only used for development/publication?

File: webmock-fixtures.gemspec (hunk 3 above)
Comment (twolfson) [inline webmock-fixtures.gemspec]: Should we open up dev dependencies to allow fluctuation on at least patch version?

``` ruby
s.add_development_dependency("rspec", [">= 3.3.0", "< 3.4.0"])
s.add_development_dependency("rubocop", [">= 0.34.2", "< 0.35.0"])
```

**Comment by twolfson** [comment]
Looks good overall. Left some questions

File: .travis.yml (hunk 1 above)
Comment (brettlangdon) [inline .travis.yml]: It isn't the default yet ;)

File: lib/webmock/fixtures/version.rb (hunk 2 above)
Comment (brettlangdon) [inline lib/webmock/fixtures/version.rb:4]: Yeah, it does seem a little weird, but this seems to be what a bunch of other gems do. `WebMock`, `factory_girl`, `rspec-core`, `HTTParty`.

File: webmock-fixtures.gemspec (hunk 3 above)
Comment (brettlangdon) [inline webmock-fixtures.gemspec]: Yeah, there is a corresponding comment in `Gemfile` to look here for dependencies.

Sure, we can open them up. No arguments.

File: .travis.yml (hunk 1 above)
Comment (twolfson) [inline .travis.yml]: Yea but there's a 3rd variation now:

http://blog.travis-ci.com/2015-10-14-opening-up-ubuntu-trusty-beta/

Comment (brettlangdon) [inline .travis.yml]: Oh man... I am behind on the times I guess.

File: webmock-fixtures.gemspec (hunk 3 above)
Comment (twolfson) [inline webmock-fixtures.gemspec]: I saw that comment. How does it play with `bundler` and `gemrat`?

File: webmock-fixtures.gemspec (hunk 4 above)
Comment (brettlangdon) [inline webmock-fixtures.gemspec:24]: Yeah, these are used by the `gem` command to package our library.

This seems to be what most of libraries I have seen done. We could instead manually list out all the files we need included in the gem. (Think python `MANIFEST.in`, but without "recursive-include").

Comment (twolfson) [inline webmock-fixtures.gemspec:24]: Ah, k. In node/npm, it defaults to `.gitignore` with an overridable `.npmignore`.

File: webmock-fixtures.gemspec (hunk 3 above)
Comment (brettlangdon) [inline webmock-fixtures.gemspec]: Should work fine. Again, this appears to be the standard for other gems. 

http://bundler.io/man/gemfile.5.html#GEMSPEC-gemspec-

`gemrat` looks just like a wrapper around `bundler` and modifying `Gemfile`, so should be good there as well.

File: lib/webmock/fixtures/version.rb (hunk 2 above)
Comment (twolfson) [inline lib/webmock/fixtures/version.rb:4]: Okie doke

**Comment by twolfson** [comment]
Looks good to me. Feel free to loosen the dev dependencies or not

**Pull request merged**